dashboard/app/controllers/pd/workshop_enrollment_controller.rb

```diff
@@ -90,43 +90,6 @@ def new
     end
   end
 
-  # POST /pd/workshops/1/enroll
```

Comment:
How did you determine this was safe to delete? I'm seeing it used in several places (I searched `/enroll` matching the word), but most importantly, here

https://github.com/code-dot-org/code-dot-org/blob/0920b732730f5c95a3b2102d2b9568c3502b6d27/apps/src/code-studio/pd/workshop_dashboard/components/workshop_enrollment.jsx#L49

Reply:
I have hit those `duplicate` and `own` partials in the past couple of months, so I'm confused about what's different now